lit_gpt/config.py

```diff
@@ -48,7 +48,7 @@ class Config:
     norm_eps: float = 1e-5
     _mlp_class: Literal["GptNeoxMLP", "LLaMAMLP"] = "GptNeoxMLP"
     intermediate_size: Optional[int] = None
-    condense_ratio: int = 1
+    rope_condense_ratio: int = 1
```

Comment:
Since we already have `rope_base` and we'll need to add one more rope-specific argument for #482, I'm clearly specifying that this refers to rope